main.py

```diff
@@ -1,10 +1,45 @@
 import init_django_orm  # noqa: F401
+import json
 
+from datetime import datetime
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as f:
+        data = json.load(f)
+
+    for nickname, player_data in data.items():
+        race_data = player_data["race"]
+        race, created = Race.objects.get_or_create(
```

Comment:
```suggestion
        race, _ = Race.objects.get_or_create(
```
You do not need info about created. In this case you can use _

Reply:
The same with guild